templates/events/manager/events/create_update.html

```diff
@@ -236,12 +233,28 @@ <h1 class="h2" tabindex="0">Create an Event</h1>
             <small class="form-text text-muted mt-0" id="help-label-tags" tabindex="-1">Provide keywords that describe this event for better searchability.</small>
           </div>
           <div class="col-md-5 col-xl-4 mb-2 mb-md-0">
-            {% render_field form.tags class="form-control" aria_describedby="help-label-tags" %}
+            <div class="row no-gutters">
+              <div class="col-10">
+                <input type="text" id="event-tags-typeahead" class="form-control" autocomplete="false">
+              </div>
+              <div class="col-2">
+                <a id="add-new-tag" href="#" class="btn btn-primary" style="display: none;">&plus;</a>
```

Comment:
We can use fontawesome icons here and on the remove button in order to match the styles of the instance remove/add buttons
```suggestion
                <a id="add-new-tag" href="#" class="btn btn-primary" style="display: none;"><span class="fa fa-plus" aria-hidden="true"></span></a>
```